demo/examples/client.rs

```diff
@@ -110,36 +111,34 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
         let m3 = IBCM::generate_issuance_m3(m1.clone(), m2.clone(), &mut rng);
         let mut m3_bytes = Vec::new();
-        m3.serialize_compressed(&mut m3_bytes).unwrap();
+        m3.serialize_compressed(&mut m3_bytes)
+            .expect("newly-generated issuance M3 should serialize");
```

Comment:
This is experimental code; error types should be converted before use in production. I left it this way for now because `ark_serialize::SerializationError` [doesn't implement](https://docs.rs/ark-serialize/latest/ark_serialize/enum.SerializationError.html) `StdError` itself, so naive `?` doesn't work.